README.md

```diff
@@ -20,13 +20,24 @@ Below table is for reference only. Measurements were made on 4C/8T x86 CPU with
 |-----------|-------------------|--------------------|----------------------|
 | `Q4_0`    | 17.507            | *76*               | **1.53**             |
 | `Q4_1`    | 17.187            | **72**             | 1.68                 |
-| `Q4_2`    | 17.060            | 85                 | **1.53**             |
 | `Q5_0`    | 16.194            | 78                 | *1.60*               |
 | `Q5_1`    | 15.851            | 81                 | 1.68                 |
 | `Q8_0`    | *15.652*          | 89                 | 2.13                 |
 | `FP16`    | **15.623**        | 117                | 2.82                 |
 | `FP32`    | **15.623**        | 198                | 5.64                 |
 
+### cuBLAS's performance on 3060Ti(8G) + i7 13700K  time cost per token
+| Model                  | Format | 24 Thread + 32 Layers on GPU | 4 Thread + 32 Layers on GPU | 1 Thread + 32 Layers on GPU |
+|------------------------|--------|------------------------------|-----------------------------|-----------------------------|
+| `RWKV-4-Raven-7B-v11`  | `Q4_0` | 100.56ms                     | 52.6ms                      | 61.6ms                      |
+| `RWKV-4-Raven-7B-v11`  | `Q4_1` | 98.4ms                       | 52.5ms                      | 63.5ms                      |
+| `RWKV-4-Raven-7B-v11`  | `Q5_1` | 137.6ms                      | 72.6ms                      | 82.6ms                      |
```

Comment:
In general, 8 CPU threads should give you the best performance for most systems

Reply:
@ggerganov seems rwkv is a lit bit different, performance of 8 threads and 2 threads are worse than 4 threads

Reply:
@ggerganov Since I focused on 7B model, it may better to 8 cpu threads on bigger model like 14b